Automattic/jetpack
# Abandon google-fonts-provider package

## Proposed changes:

The google-fonts-provider package made use of APIs that were provided by the Gutenberg plugin versions v12.8.0 to v16.3.0 and never made it into WordPress Core.

Meanwhile, the google-fonts module in the Jetpack plugin has been updated to use an API that was included in WordPress Core 6.4, which is the minimum version we support.

So let's abandon the package, and remove the back-compat code in the Jetpack plugin that used it.

### Other information:


## Jetpack product discussion


p1719259393606879-slack-CBG1CP4EN

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Downgrade WordPress to 6.4 ([which is the earliest Jetpack supports](https://github.com/Automattic/jetpack/blob/remove/abandon-google-fonts-provider-package/projects/plugins/jetpack/readme.txt#L5)).
* Make sure the Google Fonts module still works, see https://jetpack.com/support/google-fonts/ for details.
